network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractInjectionImpl.java

```diff
@@ -89,32 +87,7 @@ public <E extends Extension<I>> Collection<E> getExtensions() {
     }
 
     @Override
-    public boolean connect() {
-        return getTerminal().connect();
-    }
-
-    @Override
-    public boolean connect(Predicate<Switch> isTypeSwitchToOperate) {
-        return getTerminal().connect(isTypeSwitchToOperate);
-    }
-
-    @Override
-    public boolean connect(Predicate<Switch> isTypeSwitchToOperate, ThreeSides side) {
-        return getTerminal().connect(isTypeSwitchToOperate);
-    }
-
-    @Override
-    public boolean disconnect() {
-        return getTerminal().disconnect();
-    }
-
-    @Override
-    public boolean disconnect(Predicate<Switch> isSwitchOpenable) {
-        return getTerminal().disconnect(isSwitchOpenable);
-    }
-
-    @Override
-    public boolean disconnect(Predicate<Switch> isSwitchOpenable, ThreeSides side) {
-        return getTerminal().disconnect(isSwitchOpenable);
+    public List<Terminal> getTerminals(ThreeSides side) {
+        return (side == null || side.equals(ThreeSides.ONE)) ? Collections.singletonList(terminal) : Collections.emptyList();
```

Comment:
before we ignored the threeside argument but now we just return the empty list and the remaining code does nothing ? Is it on purpose ?

Reply:
same answer